Tools/scripts/lazy_marshal_perf.py

```diff
@@ -162,19 +164,20 @@ def get_source(self):
 def run_tests():
     results = {}
     for params in itertools.product(
-        [100],  # num_funcs
-        [100],  # func_length
-        [10, 100],  # num_vars
-        [True, False],  # is_locals
-        [True, False],  # is_unique_names
-        [True, False],  # is_vary_constants
-        [True, False],  # is_call
+        [1000],  # num_reps
+        [1000],  # num_funcs
+        [500],  # func_length
+        [10],  # num_vars
+        [False],  # is_locals
+        [True],  # is_unique_names
+        [False],  # is_vary_constants
+        [False],  # is_call
```

Comment:
Maybe don’t change these? I was hoping we’d make those command line args eventually.

Reply:
Sorry I didn't mean to change them